x-pack/plugins/secops/public/lib/ecs/index.ts

```diff
@@ -51,12 +54,25 @@ export const getMappedEcsValue = ({
 /**
  * Returns an `EcsField[]` containing all fields in the `EcsSchema`
  */
-export const allFieldsInSchema = (schema: EcsSchema): EcsField[] =>
+export const getAllFieldsInSchema = (schema: EcsSchema): EcsField[] =>
```

Comment:
I think TypeScript has advanced enough that you can use this:

```ts
export const getAllFieldsInSchema = (schema: EcsSchema): EcsField[] =>
  Object.values(schema).reduce<EcsField[]>(
    (acc, namespace) => [...acc, ...Object.values(namespace.fields)],
    []
  );
```

And remove the `as` cast.